packages/web3-shared/base/src/specs/index.ts

```diff
@@ -127,6 +127,9 @@ export enum TransactionStatusType {
     FAILED = 3,
 }
 
+/**
+ * @deprecated use TransactionStatusType instead
+ */
 export enum TransactionStateType {
```

Comment:
This is returned by the RPC method [getTransactionReceipt](https://docs.infura.io/networks/ethereum/json-rpc-methods/eth_gettransactionreceipt). We can rename it to `TransactionReceiptStatusType`.

Reply:
If we have to keep one of them, please keep `TransactionStateType`.